src/me/mrCookieSlime/Slimefun/listeners/TeleporterListener.java

```diff
@@ -63,8 +69,18 @@ else if (item.getID().equals("GPS_ACTIVATION_DEVICE_PERSONAL")) {
 			else e.setCancelled(true);
 		}
 		else if (item.getID().equals("ELEVATOR_PLATE")) {
+			if(Variables.dialogueCooldown.containsKey(e.getPlayer().getUniqueId())) return;
+			else Variables.dialogueCooldown.put(e.getPlayer().getUniqueId(), e.getClickedBlock().getLocation().toString());
 			Elevator.openDialogue(e.getPlayer(), e.getClickedBlock());
 		}
 	}
-
+	
+	@EventHandler
+	public void onPlayerMove(PlayerMoveEvent e) {
+		if(Variables.dialogueCooldown.containsKey(e.getPlayer().getUniqueId())) {
+			if(!e.getPlayer().getLocation().toString().equals(Variables.dialogueCooldown.get(e.getPlayer().getUniqueId()))) {
```

Comment:
Be very careful with this Event, it can be called much more often than you would think, for big servers it probably should not be used at all. Maybe add this check to a new **sync** task right here: https://github.com/TheBusyBiscuit/Slimefun4/blob/master/src/me/mrCookieSlime/Slimefun/SlimefunStartup.java#L357

Also the current Player Location will never match the Pressure Plate's Location. use getBlock().getLocation() instead.